models/user.go

```diff
@@ -826,10 +828,10 @@ func ChangeUserName(u *User, newUserName string) (err error) {
 	if err = x.
 		Where("owner_id=?", u.ID).
 		Iterate(new(Repository), func(idx int, bean interface{}) error {
-			repo := bean.(*Repository)
-			RemoveAllWithNotice("Delete repository wiki local copy", repo.LocalWikiPath())
-			return nil
-		}); err != nil {
+		repo := bean.(*Repository)
+		RemoveAllWithNotice("Delete repository wiki local copy", repo.LocalWikiPath())
+		return nil
+	}); err != nil {
```

Comment:
If I read correctly this is an indent-only change, and is also introducing a wrong indent (does "make lint" not complain about it?)